src/deploy.sh

```diff
@@ -1006,13 +1006,6 @@ function run_kernel_install()
   name=${name:-'kw'}
   flag=${flag:-'SILENT'}
 
-  if [[ "$reboot" == 0 ]]; then
-    reboot_default="${deploy_config[reboot_after_deploy]}"
-    if [[ "$reboot_default" =~ 'yes' ]]; then
-      reboot=1
-    fi
-  fi
```

Comment:
Maybe I miss something, but I still want to read the config file to get the default behavior for the reboot option. What I expect from the `--reboot` or `--no-reboot` is that those options take high priority over the config file. For example:
* deploy.config set reboot by default
* kw deploy -> it will reboot after deploy
* kw deploy --no-reboot -> it will not reboot, even if the config file set reboot as the default behavior.